src/flatpak-spawn.c

```diff
@@ -853,14 +949,38 @@ main (int    argc,
 
       check_portal_version ("sandbox-expose-path-ro", 3);
 
-      if (!add_paths_to_variant (expose_fd_builder, fd_list, opt_sandbox_expose_path_ro, FALSE)
-          || !add_paths_to_variant (expose_fd_builder, fd_list, opt_sandbox_expose_path_ro_try, TRUE))
+      if (!add_paths_to_variant (expose_fd_builder, fd_list, opt_sandbox_expose_path_ro, home_realpath, flatpak_id, FALSE)
+          || !add_paths_to_variant (expose_fd_builder, fd_list, opt_sandbox_expose_path_ro_try, home_realpath, flatpak_id, TRUE))
         return 1;
 
       g_variant_builder_add (&options_builder, "{s@v}", "sandbox-expose-fd-ro",
                              g_variant_new_variant (g_variant_builder_end (g_steal_pointer (&expose_fd_builder))));
     }
 
+  if (opt_runtime_usr != NULL)
+    {
+      gint32 handle;
+
+      g_debug ("Using %s as /usr instead of runtime", opt_runtime_usr);
+
+      if (opt_host)
+        {
+          g_printerr ("--host not compatible with --runtime-usr\n");
+          return 1;
+        }
+
+      /* TODO: it'll probably really be version 4 */
+      check_portal_version ("runtime-usr-fd", 3);
```

Comment:
This should be updated to whatever version of the portal interface merges flatpak#4018, but I don't think it's appropriate to set this until that feature is basically ready for merge.

Reply:
Now assumed to be version 6, which matches the latest revision of flatpak/flatpak#4018.

Reply:
flatpak/flatpak#4018 was merged, and version 6 is indeed the correct version.